### PR TITLE
Validate form

### PR DESCRIPTION
**Pull request by Pamphilemkp**
form validation added 
when click on the submit button checks for  validation if valide the form will be submitted if not will throw an error and highlight the error type and label.
form validation with javascript

### CODE REVIEW

**Comment by AzizRAA** [comment]
Hello, I'm sorry I left a message for you before sending this project for review because there's only one linter error. It's a Web hint, I already reached out to Student Succes and they recommend ignoring it since it's not relevant for this project, kindly check oject and the details of the error again, please.
image
image

File: index.js [hunk 1]
```diff
@@ -11,4 +11,118 @@ function menubar() {
...
+
+  // email validation
+  if (emailValue === '') {
+    setErrorEmail(email, 'Email is required');
+    return;
+  }
+  if (emailValue !== emailValue.toLowerCase()) {
+    setErrorEmail(email, 'Email should be in lowercase');
+    return;
+  }
```
Comment (tresorsawasawa) [inline index.js:92]: - [ ] Good job with your email validation, it works as expected. But according to the project's requirements, the error message is supposed to show up close to the "submit" button i.e. **Get In Touch.** button. Kindly make sure you improve its styling and put it displaying on the **Get In Touch.** button and not at the top of the form.

File: index.html [hunk 2]
```diff
@@ -339,11 +339,15 @@ <h3>Contact me</h3>
...
-    <input type="text" name="name" placeholder="ex: pamphile" maxlength="30" required> <br>
-    <input type="email" name="email" placeholder="ex: pamphilemkp@gmail.com" maxlength="30" required> <br>
-    <textarea placeholder=" Write your message here" name="comment" id="comment" cols="30" rows="10" required maxlength="500"></textarea><br>
+  <form id="form" action="https://formspree.io/f/xyyovzvy" method="post">
+    <p class="Paragraph-errors errorMessage"></p>
+    <input type="text" name="name" id="name" placeholder="ex: pamphile" maxlength="30" > <br>
+    <input type="email" name="email" id="email" placeholder="ex: pamphilemkp@gmail.com" maxlength="30" >
+    <br>
+    <textarea placeholder=" Write your message here" id="comment" name="comment" cols="30" rows="10"  maxlength="500"></textarea><br>
+
```
Comment (tresorsawasawa) [inline index.html]: - [ **OPTIONAL** ] **Important:** Kindly make sure you update the message text color in the textarea field by using the same(color) as on the Figma design for good accessibility and to match the Figma template. I know you can do it :muscle: :muscle:.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Pamphilemkp,
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle:
- [ ] **Kindly make sure you added all the **authors' contributors** with their social links in your `README` file and if possible improve the `project's description` and other sections to make it professional. I suggest you use this [README](https://github.com/tresorsawasawa/MyPortfolio#readme) as the template.
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Pamphilemkp 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**